resources/azure/environment.go

```diff
@@ -97,3 +98,25 @@ func (e *Environment) DefaultPrivateKeyPassword() string {
 func (e *Environment) GetCommonEnvironment() *config.CommonEnvironment {
 	return e.CommonEnvironment
 }
+
+// ContainerRuntime Determines the type of workload a node can run.
+// Note that KataMshvVmIsolation is supported only on Mariner Sku.
+// Allowed values: KataCcIsolation, KataMshvVmIsolation, OCIContainer, WasmWasi.
+// Default : OCIContainer.
+func (e *Environment) ContainerRuntime() string {
+	env := e.CommonEnvironment.ContainerRuntime()
+	switch env {
+	case "kata":
```

Comment:
IMO you can define a `const string` for the value `kata`

Reply:
Done